stages/org.osbuild.oscap.remediation

```diff
@@ -61,63 +63,76 @@ SCHEMA = """
         "type": "string",
         "description": "The tailoring id"
       },
-      "xml_results": {
+      "arf_results": {
         "type": "string",
-        "description": "Filename for saving the xml result file",
-        "default": "eval_remediate_results.xml"
+        "description": "Filename for storing the ARF Results file"
       },
-      "arf_results": {
+      "xccdf_results": {
         "type": "string",
-        "description": "Filename for saving the arf result file"
+        "description": "Filename for storing the XCCDF Results file"
       },
       "html_report": {
         "type": "string",
-        "description": "Filename for saving the html report",
-        "default": "eval_remediate_report.html"
+        "description": "Filename for saving the final HTML report"
       },
       "verbose_log": {
         "type": "string",
         "description": "Filename for verbose error messages"
       },
       "verbose_level": {
         "type": "string",
-        "enum": ["DEVEL", "INFO", "ERROR", "WARNING"],
+        "enum": ["DEVEL", "INFO", "WARNING", "ERROR"],
         "description": "The verbosity level for the log messages"
+      },
+      "compress_results": {
+        "type": "boolean",
+        "default": false,
+        "description": "Compress ARF and XCCDF results file(s) with xz"
       }
     }
   }
 }
 """
 
-XML_RESULTS = "eval_remediate_results.xml"
-HTML_REPORT = "eval_remediate_report.html"
+DATA_DIR = "/root"
+XCCDF_RESULTS = "oscap_eval_xccdf_results.xml"
```

Comment:
please note this default in the stage schema

Reply:
Done.